pkg/cmd/taskrun/list_test.go

```diff
@@ -168,6 +169,18 @@ func TestListTaskRuns(t *testing.T) {
 			args:      []string{"list", "-n", "invalid"},
 			wantError: true,
 		},
+		{
+			name:      "filter pipelineruns by label",
+			command:   command(t, trs, now, ns),
+			args:      []string{"list", "-n", "foo", "--labels", "honey=pot"},
+			wantError: false,
+		},
+		{
+			name:      "no mixing pipelinename and labels",
+			command:   command(t, trs, now, ns),
+			args:      []string{"list", "-n", "foo", "--labels", "viva=wakanda", "tr3-1"},
+			wantError: true,
+		},
```

Comment:
Can you add a test for multiple labels?

Reply:
Good idea 👍

Reply:
intereslingly what is documented (from kubectl) doesnt work 😢  : 

```
% tkn tr ls --labels tekton.dev/task==pipeline-test,tekton.dev/task==build-tektoncd-pipeline-and-pushb
[nothing]
```

but this weird syntax does : 

```
% tkn tr ls --labels 'tekton.dev/task in (pipeline-test,build-tektoncd-pipeline-and-push)'
NAME                                         STARTED     DURATION     STATUS
pipeline-test-run-ddg7g                      1 day ago   30 minutes   Succeeded
build-tektoncd-pipeline-and-push-run-h6cdc   1 day ago   6 minutes    Succeeded
```

Reply:
does not seems to work since that's service side sutff, so I guess we are kept at bay with only a single label filtering (unless we get the whole list which could get really slow)